projects/plugins/jetpack/modules/shortcodes/gravatar.php

```diff
@@ -93,16 +98,60 @@ function jetpack_gravatar_profile_shortcode( $atts ) {
 		return false;
 	}
 
+	$hashed_email = hash( 'sha256', strtolower( trim( $user->user_email ) ) );
+	$cache_key    = 'grofile-' . $hashed_email;
```

Comment:
Do you think we could use a `jetpack_` prefix transient name here, so folks looking at transients on their site immediately know where it is set? `grofile` may be a bit obscure for folks not familiar with the shortcode.